ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -88,11 +93,16 @@ extension TableViewHeaderFooterViewDemoController: UITableViewDelegate {
 
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: TableViewHeaderFooterView.identifier) as! TableViewHeaderFooterView
+        let index = section
         let section = tableView.style == .grouped ? groupedSections[section] : plainSections[section]
+        if section.hasHandler {
+            header.onHeaderViewTapped = { [unowned self] in self.forHeaderTapped(header: header, section: index) }
```

Comment:
I think it's safer (best practice) to use weak instead of unowned and make the call like this:

self?.forHeaderTapped(header: header, section: index)